src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -485,6 +487,14 @@
       <LogicalName>System.Data.SqlClient.SqlMetaData.xml</LogicalName>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup  Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
You will want to condition the netcoreapp specific APIs to only be included in the netcoreapp build configuration.